import-export-cli/impl/exportAPIProduct.go

```diff
@@ -98,6 +98,7 @@ func WriteAPIProductToZip(exportAPIProductName, exportAPIProductVersion, zipLoca
 				ImportAPIs:       true,
 				UpdateAPIProduct: true,
 				UpdateAPIs:       false,
+				RotateRevision:   true,
```

Comment:
We shouldn't add rotate revision by default to any place.

Reply:
Yes, agree with you. Will change the default value to false here as well. So when populating the api_meta.yaml, it will have rotateRevision: false